acquisitions/src/main/resources/thunderjet/mod-orders/features/independent-acquisitions-unit-for-ordering-and-receiving.feature

```diff
@@ -0,0 +1,720 @@
+
+    # 4. Create a composite order
+    * configure headers = headersUser
+    Given path 'orders/composite-orders'
+    And request
+    """
+    {
+      id: '#(orderId1)',
+      vendor: '#(globalVendorId)',
+      orderType: 'One-Time',
+      "acqUnitIds": ['#(acqUnitId1)']
+    }
+    """
+    When method POST
+    Then status 201
+
+    # 5. Create an order line
+    Given path 'orders/order-lines'
+    * def poLine = read('classpath:samples/mod-orders/orderLines/minimal-order-line.json')
+    * set poLine.id = poLineId1
+    * set poLine.purchaseOrderId = orderId1
+    * set poLine.fundDistribution[0].fundId = fundId
+    And request poLine
+    When method POST
+    Then status 201
+
+    # 6. Open the order
+    Given path 'orders/composite-orders', orderId1
+    When method GET
+    Then status 200
+
+    * def order = $
+    * set order.workflowStatus = 'Open'
+
+    Given path 'orders/composite-orders', orderId1
+    And request order
+    When method PUT
+    Then status 204
+
+    # 6. Check that acqUnit1 was inherited from Order to Title
+    * configure headers = headersUser
+    Given path 'orders/titles'
+    And param query = 'poLineId==' + poLineId1
+    When method GET
+    Then status 200
+
+    And match $.totalRecords == 1
+
+  Scenario: Edit acq units for Order, change should NOT propagated to related Title
+    # 1. Edit the composite orde
+    * configure headers = headersUser
+    Given path 'orders/composite-orders', orderId1
+    When method GET
+    Then status 200
+
+    * def order = $
+    * set order.acqUnitIds = []
+
+    Given path 'orders/composite-orders', orderId1
+    And request order
+    When method PUT
+    Then status 204
+
+    # 2. Check title has acqUnit
+    Given path 'orders/titles'
+    And param query = 'poLineId==' + poLineId1
+    When method GET
+    Then status 200
+    And match $.totalRecords == 1
+    And match $.titles[0].acqUnitIds[0] == acqUnitId1
+
+  Scenario: Edit acq units for Title, change should NOT propagated to related Order
+    # 1. Assign acqUnit to order back
+    * configure headers = headersUser
+    Given path 'orders/composite-orders', orderId1
+    When method GET
+    Then status 200
+
+    * def order = $
+    * set order.acqUnitIds = []
+
+    Given path 'orders/composite-orders', orderId1
+    And request order
+    When method PUT
+    Then status 204
+
+    # 2. Remove acqUnit from title
+    Given path 'orders/titles'
+    And param query = 'poLineId==' + poLineId1
+    When method GET
+    Then status 200
+    And match $.totalRecords == 1
+    * def titleResponse = $.titles[0]
+    * set titleResponse.acqUnitIds = []
+    * def titleId = $.titles[0].id
+
+    Given path 'orders/titles', titleId
+    And request titleResponse
+    When method PUT
+    Then status 204
+
+    # 3. Check that order has acqUnitIds
+    Given path 'orders/composite-orders', orderId1
+    When method GET
+    Then status 200
+    And match $.orders[0].acqUnitIds == acqUnitId1
```

Comment:
previously in the scenario you set order.acqUnitIds = [], so after updating title acqUnits - order acqUnits should not be changed and equals to empty array as well

Reply:
fixed